webapi/src/test/scala/org/knora/webapi/e2e/v2/OntologyV2R2RSpec.scala

```diff
@@ -13,6 +13,7 @@ import org.knora.webapi.messages.store.triplestoremessages.RdfDataObject
 import org.knora.webapi.messages.util.rdf._
 import org.knora.webapi.messages.v2.responder.ontologymessages.{InputOntologyV2, TestResponseParsingModeV2}
 import org.knora.webapi.messages.{OntologyConstants, SmartIri, StringFormatter}
+import org.knora.webapi.models._
```

Comment:
Is this really needed? I don't see any newly added code in this file but this import statement.

Reply:
I moved OntologyModels to the models package, which previousely was in the same package as this file. Therefor calls to the classes in OntologyModel have remained unchanged (so no changed code), but I needed to add the import to be able to make those calls.